spec/app/dumpers/z_after_hook_dumper.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+FILENAME = 'tmp/test.out'
+
+# Filename prepended with z to so it is run last
+ResponseDumper.define 'after_hook' do
+  before do
+    FileUtils.rm_rf(FILENAME)
+  end
```

Comment:
```suggestion
```

Reply:
I added this before block because the presence of a file from a previous test run obscured a bug I had introduced to the after hook execution when testing locally. Is there a better way to ensure I'm writing a file each time tests are run?

Reply:
I'm suggesting we use a unique temporary directory per run. I put some details in comment: https://github.com/Pioneer-Valley-Books/rails-response-dumper/pull/40#discussion_r997405059

We can then also ensure the temporary files get cleaned up after a test run.